code/presentation/mb-client/src/screens/home/find-masjid.tsx

```diff
@@ -0,0 +1,184 @@
+import { useNavigation } from '@react-navigation/native';
+import { FlashList } from '@shopify/flash-list';
+import * as Location from 'expo-location';
+import React, { useState } from 'react';
+import { Button, StyleSheet, TextInput } from 'react-native';
+import type { Details, Region } from 'react-native-maps';
+import MapView, { enableLatestRenderer, Marker } from 'react-native-maps';
+
+import type { MasjidWithDistance, NamazTimings } from '@/api/masjid/types';
+import { useMasjids } from '@/api/masjid/use-masjids';
+import { getNextNamaz } from '@/api/masjid/util';
+import { Text, TouchableOpacity, View } from '@/ui';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  map: {
+    flex: 1,
+  },
+  mapContainer: {
+    flex: 1,
+  },
+});
```

Comment:
add a comment saying that, we used css as native wind doesn't work with map, if that is the case